### ISSUE TITLE
"servers transport not found default@internal" error when using file provider

### ISSUE THREAD

**Issue by aku** [traefik/traefik#7809]
Traefik shows following error when using file provider but continue to work properly

```
ERRO[2021-01-27T19:42:10+03:00] servers transport not found default@internal  entryPointName=web routerName=load-balancing-demo@file
servers transport not found
```

I'm using Traefik version 2.4.0 installed via brew (also tried to download a binary from the official site)
OS: MacOS BigSur 11.0.1 (20B29)

Command used to run Traefik
```
traefik --configfile demo.yaml
```

Static config demo.yaml

```yaml
entryPoints:
  web:
    address: ":80"
  traefik:
    address: ":9000"

log: {}

accessLog: {}

api:
  insecure: true
  dashboard: true

metrics:
  prometheus: {}

providers:
  file:
    filename: load-balancing.yaml
    watch: true
```

Dynamic config load-balancing.yaml

```yaml
http:
  routers:
    load-balancing-demo:
      entryPoints:
        - web
      service: load-balanced-svc
      rule: Host(`demo.local`)

  services:
    load-balanced-svc:
      loadBalancer:
        servers:
          - url: localhost:3001
```

**Comment by lwis** [comment]
Is this somethijng to be concerned about?

**Comment by aku** [comment]
> Is this something to be concerned about?

This is a bug introduced in 2.4 - default@internal round tripper is never created. I'm creating a series of tutorials on Traefik and having this error with a perfectly valid config file is certainly making a bad impression of the product.

@juliens can you check please? Looks like this bug was introduced in [PR 7203](https://github.com/traefik/traefik/pull/7203)

**Comment by voodooattack** [comment]
I'm having the same issue. Just started integrating 2.4 into my project and was getting this error when trying to using the file provider. Downgrading to 2.3 fixed it for me.

**Comment by traefiker** [comment]
Closed by #7839.
